### PR TITLE
added debug from visionmedia

### PR DESCRIPTION
**Pull request by capaj**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -132,6 +132,7 @@
   "d3": "github:mbostock/d3",
   "datatables": "github:DataTables/DataTables",
   "decimal.js": "github:MikeMcl/decimal.js",
+  "debug": "github:visionmedia/debug",
```
Comment (guybedford) [inline registry.json]: To be pedantic, should be one line up.

**Comment by capaj** [comment]
@guybedford you would spot typo a mile away, wouldn't you :smile:

**Comment by crissdev** [comment]
@capaj I guess squashing the commits into one would be great.

**Comment by guybedford** [comment]
Thanks, sorry for the delay!

**Pull request merged**